command.go

```diff
@@ -3,7 +3,7 @@
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-// http://www.apache.org/licenses/LICENSE-2.0
+// https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Official Apache licenses use `http` - please revert this change.

Ref: https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
Change was reverted as requested.